test_cases/labels.json

```diff
@@ -232,6 +232,140 @@
           "label": "People's Republic of China"
         }]
       }
+    },
+    {
+      "id": 17,
+      "status": "pass",
+      "description": "locality in US dependency should have dependency as least granular field",
+      "issue": "https://github.com/pelias/api/issues/628",
+      "user": "trescube",
+      "in": {
+        "text": "bayamon, pr",
+        "sources": "wof",
+        "layers": "locality"
+      },
+      "expected": {
+        "properties": [
+          {
+            "label": "Bayamón, PR"
+          }
+        ]
+      }
+    },
+    {
+      "id": 18,
+      "status": "pass",
+      "description": "locality in US dependency should have dependency as least granular field",
+      "issue": "https://github.com/pelias/api/issues/628",
+      "user": "trescube",
+      "in": {
+        "text": "aasu, as",
+        "sources": "wof",
+        "layers": "locality"
+      },
+      "expected": {
+        "properties": [
+          {
+            "label": "Aasu, AS"
+          }
+        ]
+      }
+    },
+    {
+      "id": 19,
+      "status": "pass",
+      "description": "locality in non-US dependency should still include country",
+      "issue": "https://github.com/pelias/labels/issues/5",
+      "user": "trescube",
+      "in": {
+        "text": "George Hill, Anguilla",
+        "sources": "wof",
+        "layers": "locality",
+        "boundary.country": "GBR"
+      },
+      "expected": {
+        "properties": [
+          {
+            "label": "George Hill, United Kingdom"
+          }
+        ]
+      }
+    },
+    {
+      "id": 20,
+      "status": "pass",
+      "description": "locality in non-US dependency should still include country",
+      "issue": "https://github.com/pelias/labels/issues/5",
+      "user": "trescube",
+      "in": {
+        "text": "Tórshavn faroe islands",
+        "sources": "wof",
+        "layers": "locality"
+      },
+      "expected": {
+        "properties": [
+          {
+            "label": "Tórshavn, Denmark"
```

Comment:
We eventually want this label to be something like "Tórshavn, Fareo Islands", right? (just confirming my understanding)